components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -3333,6 +3333,17 @@ func TestCreateRule(t *testing.T) {
 			_, err := storage.NewRule("new-id-1", projID, "name", storage.Node, []storage.Condition{})
 			assert.Error(t, err)
 		},
+		"creating a condition with zero entries returns an error": func(t *testing.T) {
+			condition1, err := storage.NewCondition(storage.Event, []string{}, storage.ChefServer, storage.Equals)
+			assert.Equal(t, storage.Condition{}, condition1)
+			assert.Error(t, err)
+		},
+		"creating an equals condition with multiple entries returns an error": func(t *testing.T) {
+			condition1, err := storage.NewCondition(storage.Event,
+				[]string{"chef-server-1", "chef-server-2"}, storage.ChefServer, storage.Equals)
+			assert.Equal(t, storage.Condition{}, condition1)
+			assert.Error(t, err)
+		},
```

Comment:
Suggest a couple more tests:
"multiple entries for a 'member-of' condition are allowed"
"single entry for a 'member-of' condition is allowed"

Reply:
We have a million tests that cover this case but I can add them explicitly.